moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -522,7 +522,10 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
 
   bool setPoseTargets(const std::vector<geometry_msgs::PoseStamped>& poses, const std::string& end_effector_link)
   {
+      ROS_ERROR_STREAM_NAMED("move_group_interface","end_effector_link_" << end_effector_link_ << "\n");
+      ROS_ERROR_STREAM_NAMED("move_group_interface","end_effector_link" << end_effector_link << "\n");
```

Comment:
extra indentation here. also, the "\n" will cause double line breaks, which is not necessary.

also - the line is repeated twice?